packages/p2p/src/pages/advertiser-page/advertiser-page-dropdown-menu/advertiser-page-dropdown-menu.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Dropdown, Icon, useOnClickOutside } from '@deriv/components';
+import { observer } from '@deriv/stores';
+import { localize } from 'Components/i18next';
+import { useStores } from 'Stores/index';
+
+const AdvertiserPageDropdownMenu = () => {
+    const dropdown_menu_ref = React.useRef<HTMLDivElement>(null);
+    const { advertiser_page_store } = useStores();
+    const {
+        is_counterparty_advertiser_blocked,
+        is_dropdown_menu_visible,
+        setIsDropdownMenuVisible,
+        showBlockUserModal,
+    } = advertiser_page_store;
+
+    const onClickOutside = () => {
+        advertiser_page_store.setIsDropdownMenuVisible(false);
```

Comment:
can remove the advertiser_page_store in front since you've already destructured it

```suggestion
        setIsDropdownMenuVisible(false);
```